resources/keywords/navigation.resource

```diff
@@ -47,24 +47,40 @@ User Navigates To DBaaS Inventories Screen
     Wait Until Page Contains Element    ${elemDbaasStatus}    timeout=15s
     Page Should Contain Element    ${lnkProvAcc}
     Click Element    ${lnkProvAcc}
-    Wait Until Page Contains Element    ${titleDbaasInventories}    timeout=10s
 
 User Navigates To DBaaSPolicy Screen
-    [Arguments]    ${project}=${operatorNamespace}
+    [Arguments]    ${PROJECT}=${operatorNamespace}
     User Selects Administrator View
     User Navigates To Installed Operators Under Operators
-    User Filters ${project} Namespace On Project Dropdown
+    User Filters ${PROJECT} Namespace On Project Dropdown
     Wait Until Page Contains Element    ${elemDbaasStatus}    timeout=15s
     Wait Until Keyword Succeeds    5x    3s    Run Keywords
     ...    User Navigates To Database Access Under Data Services
     ...    AND    User Navigates To Installed Operators Under Operators
     ...    AND    Element Should Be Visible    ${lnkDBaaSPolicy}    timeout=10s
     Click Element    ${lnkDBaaSPolicy}
-    Wait Until Page Contains Element    ${titleDbaasPolicies}    timeout=10s
+    ${version}=     Run Keyword And Return Status    Wait Until Page Contains Element    ${titleDbaasPolicies}    timeout=10s
+    IF  not ${version}
+        Element Should Be Visible    ${titleDbaasPolicys}
+        Log    Cluster version is 4.11, Known issue
+    ELSE
+        Fail    msg=DBaaSPolicy page not loaded
+    END
 
 User Filters ${projectName} Namespace On Projects Screen
     Wait Until Page Contains Element    ${txtProjects}    timeout=10s
     Element Should Be Visible    ${txtBxSearchByName}
     Input Text    ${txtBxSearchByName}    ${projectName}
     ${lnkProjectNameUpd} =    Replace String    ${lnkProjectName}    <<project>>    ${projectName}
     Wait Until Page Contains Element    ${lnkProjectNameUpd}    timeout=5s
+
+User Is On DBaaSInventory Screen
+   ${version_410}=     Run Keyword And Return Status    Wait Until Page Contains Element    ${titleDbaasInventories}    timeout=10s
+    IF  not ${version_410}
+        ${version_411}=     Run Keyword And Return Status    Wait Until Page Contains Element    ${titleDbaasInventorys}    timeout=10s
+        IF    ${version_411}
```

Comment:
Why do we need this IF? Even if it is not 4.11 it is probably >=4.11. Maybe we can change the parameter ${version_411} to ${version_411_or_more} and remove the second IF?

Reply:
The flow is kind of we are looking for ${titleDbaasInventories} first, if it is available we would be ending the conditional statement. But if it is not available then we cannot directly say the version is 4.11.
That's the reason for second IF. So, we are verifying ${titleDbaasInventorys} is present and visible then only we are confident to say the cluster is on version 4.11.
And I am sticking with 4.11 for now for the variable naming as the higher versions are not explored.

Reply:
I see what you mean. But in the future, Until they fix this (and it does not seem to be fixed soon), I think that any version >4.10 is a version that is going to apply to the first IF. We can leave it for now but in versions >=4.12, if the bug will still exist, we cannot go on with this, and we should remove the second IF (Any version>=4.10 has the bug as long as it is not fixed). LMKWYT

Reply:
OK